verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -39,8 +39,10 @@ export default class CasesController {
     };
 
     /** Download forwards the request to the data service and streams the
-     * streamed response as a csv attachment. */
+     * streamed response as an attachment. */
     download = async (req: Request, res: Response): Promise<void> => {
+        const correlationId = crypto.randomBytes(16).toString('hex');
+        req.body.correlationId = correlationId;
```

Comment:
Won't this override the correlationId set by downloadAsync? unless that's the intended behaviour

Reply:
I don't think so; it's a separate endpoint from `downloadAsync` and one does not redirect to the other